build.sh

```diff
@@ -45,18 +46,21 @@ openslide_name="OpenSlide"
 openslidejava_name="OpenSlide Java"
 
 # Package versions (omit Meson packages)
-openslide_ver="3.4.1"
+#openslide_ver="3.4.1"
+openslide_ver="libdicom-test-release"
```

Comment:
This shouldn't be necessary.  For local testing you can use the [override directory](https://github.com/openslide/openslide-winbuild#substitute-sources), and I'm not so concerned about CI-testing this PR with a libdicom-enabled OpenSlide.

Reply:
OK, reverted.